readme.md

```diff
@@ -1,2 +1,393 @@
+  </dd>
+</dl>
+
+[Slides](http://fr.slideshare.net/denderello/symfony-live-2010-using-doctrine-migrations)  
+~~Video~~
+
+By [Dennis Benkert](https://connect.sensiolabs.com/profile/denderello)  
+![github](icon/github.png) [@denderello](https://github.com/denderello)  
+![twitter](icon/twitter.png) [@denderello](https://twitter.com/denderello)
+
+---
+
+## Working with the admin generator
+
+<dl>
+  <dt>Description</dt>
+  <dd>
+    Present some real life experiences concerning the admin generator. The main content is the presentation
+    of how to add extra functionality and discussions on how far it should be pushed.
+  </dd>
+</dl>
+
+[Slides](http://fr.slideshare.net/denderello/symfony-live-2010-using-doctrine-migrations)  
+~~Video~~
+
+By [John Cleveley](https://connect.sensiolabs.com/profile/jcleveley)  
+![github](icon/github.png) [@jcleveley](https://github.com/jcleveley)  
+![twitter](icon/twitter.png) [@jcleveley](https://twitter.com/jcleveley)
+
+---
+
+## The symfony community - How you can (get) help
+
+<dl>
+  <dt>Description</dt>
+  <dd>
+    Any community is about a good balance of giving and taking and in the symfony
+    community there is no difference. During this talk, this balance between giving
+    and taking in the symfony community will be discussed. Both for individual developers
+    and for companies, this presentation will show you ways of getting help from the community
+    as well as contributing to the community. At the end of the presentation, you will be able
+    to create a healthy balance between the two.
+  </dd>
+</dl>
+
+[Slides](http://fr.slideshare.net/skoop/the-symfony-community-how-to-get-help)  
+~~Video~~
+
+By [Stefan Koopmanschap](https://connect.sensiolabs.com/profile/skoop)  
+![github](icon/github.png) [@skoop](https://github.com/skoop)  
+![twitter](icon/twitter.png) [@skoop](https://twitter.com/skoop)
+
+---
+
+## An offline admin generator with HTML 5 and Gears
+
+<dl>
+  <dt>Description</dt>
+  <dd>
+    Symfony does a lot of things, so as your admin-generator. However,
+    we still need to be connected to our app to use them. Gears, and now
+    HTML 5, makes our web pages offline capable, so why not our
+    admin-generators? We'll see how to contribute to your app from a
+    submarine or a cave and how to improve online performances with
+    local caching. The conference will be the occasion for the release
+    of a Gears-Admin generator plugin.
+  </dd>
+</dl>
+
+[Slides](http://fr.slideshare.net/oncletom/an-offline-admin-generator-with-html-5-and-gears)  
+~~Video~~
+
+By Thomas Parisot
+
+---
+
+## Internationalization
+
+<dl>
+  <dt>Description</dt>
+  <dd>
+    Internationalization is an important feature in a worldwide project.
+    All-important projects required at some point to be used by people
+    with different cultures and languages. This constraint must be
+    understood from the application architecture to the implementation.
+  </dd>
+</dl>
+
+~~Slides~~  
```

Comment:
http://fr.slideshare.net/th0masr/internationalization-with-the-symfony-framework